serverlessworkflow/modules/ROOT/pages/integrations/camel-routes-integration.adoc

```diff
@@ -0,0 +1,133 @@
+
+.Procedure
+. To add the Quarkus Camel to your workflow application, add the `org.kie.kogito:kogito-addons-quarkus-camel` dependency to the `pom.xml` file of your project as follows:
++
+--
+.Dependency to be added to the `pom.xml` file to enable metrics
+[source,xml]
+----
+<dependency>
+    <groupId>org.kie.kogito</groupId>
+    <artifactId>kogito-addons-quarkus-camel</artifactId>
+</dependency>
+----
+--
+
+[[con-creating-camel-routes]]
+== Creating Camel routes in {context}
+
+You can add YAML or XML Camel routes to your workflow project.
+
+.Procedure
+. Create a YAML or XML Camel Routes using your IDE or the link:{kaoto_url}[Kaoto VSCode Editor] and place them in the `src/main/resources/routes` directory.
+. The route `from` endpoint must be a `direct` component. That's the endpoint producer expected by the workflow engine.
+. The route response must be in a valid format that the workflow context can understand:
++
+include::../../pages/_common-content/camel-valid-responses.adoc[]
++
+The response will be merged into the workflow state context. If it is an array or a complex object, the response will be added to the special attribute `response`.
+
+.Example of a Camel route with the `direct` endpoint returning a valid JSON string representation
+[source,yaml]
+----
+- from:
+    uri: direct:logRouteReplaceHeader
+    steps:
+      - setBody:
+          simple: '{ "id": "${header.WorkflowID}", "arg1": { "arg2": "value1" } }'
+      - log:
+          message: We received the ${body}
+----
+
+[[con-importing-camel-routes]]
+== Defining and referencing Camel functions in the Workflow DSL
+
+You can define and reference your Camel functions in the workflow definition.
+
+.Prerequisites
+* You have created Camel routes in the workflow Maven project.
+
+.Procedure
+. In the `functions` definition section of your workflow DSL, declare the Camel route as exemplified below:
++
+.Example of a Camel Route function definition
+[source,json]
+----
+{
+  "functions": [
+    {
+      "name": "logRoute",
+      "type": "custom",
+      "operation": "camel:direct:logRouteReplaceHeader"
+    }
+  ]
+}
+----
++
+The operation description must have the prefix `camel:direct:`, indicating that you want to produce a message to this route via the link:{camel_extensions_url}/direct.html[Camel Direct Component]. Direct is the only component supported by {context} at the moment.
++
+The `operation` suffix contains the name of the route endpoint. In the case of this example, `logRouteReplaceHeader`.
+
+[WARNING]
+====
+The Camel route defined in the workflow must be available in your project during runtime, otherwise, an `IllegalArgumentException` will be thrown.
+====
+
+. To use the Camel function definition in a workflow action, you can simply reference it as you normally would with any other {context} function definitions. For example:
```

Comment:
numbering here is wrong in the demo from netlify

![image](https://user-images.githubusercontent.com/17157711/208699701-5c0c893c-5fbc-4c58-a594-84e69627f40c.png)

It should be `2.`